### DISCUSSION TITLE
Weboberfläche läd nur noch sporadisch (wenn überhaupt)

### DISCUSSION THREAD

**Discussion by nehruchris** [jomjol/AI-on-the-edge-device#2844] [Q&A]
Infos:
**Release:** v15.4.0 (Commit: 74d4f20)
**SD Karte:** 16GB, welche bisher problemlos funktioniert hat. Nun habe ich eine 32 GB Sandisk (neu) gekauft, womit ich aber das selbe Verhalten habe. (habe auch mal eine 128GB mit fat32 formatiert ausprobiert aber die schneit der ESP nicht lesen zu können)
**Netzteil:** Das Netzteil mit dem es bisher lief ist immernoch funktionstüchtig. Habe aber 2x weitere durchprobiert ohne Erfolg.
**WLAN:** keine Veränderung getätigt

Ich hab den ESP32 mit dem ESP Flash tool erased und die 15.4 neu flashed.

Das Verhalten kann wie folgt beschrieben werden:
Alles lief gute ca. 6 Monate lang mit dem vorherigen Release einwandfrei, bis sich das Teil irgendwann nur noch sporadisch und dann irgendwann mal gar nicht mehr Daten zum MQTT gemeldet hat. Andere Devices melden nachwievor Daten zum MQTT, der kann also auch ausgeschlossen werden. Die neue FW habe ich mit OTA erfolgreich geflashed gehabt.
Die Oberfläche war, so wie die Daten beim MQTT, auch nur noch sporadisch erreichbar und hat auch ewigkeiten geladen.
Momentan warte ich mehrere Minuten bis die Website geladen hat. Manchmal läd sie auch gar nicht fertig. Auf klick Aktionen in der Weboberfläche reagiert das Teil auch nur noch selten.
Aktuell bekomme ich noch nichtmal den Einrichtungsassi fertiggestellt.

Ich bin langsam ratlos und haue gleich mit dem Hammer auf dem Teil rum !

Ich wäre so dankbar über einen funktionierenden AI on the Edge....

Grüße,
Chris

**Comment by nehruchris** [comment]
Eigentlich habe ich ein ähnliches Verhalten wie der Kollege in dem [Post ](https://github.com/jomjol/AI-on-the-edge-device/discussions/2406)hier

**Reply by Slider0007** [reply]
Hier ein paar Ideen: https://github.com/jomjol/AI-on-the-edge-device/discussions/2406#discussioncomment-8348127
